lib/Remotery.c

```diff
@@ -4369,6 +4382,14 @@ static void SetDebuggerThreadName(const char* name)
     #else
         RMT_UNREFERENCED_PARAMETER(name);
     #endif
+
+    #ifdef RMT_PLATFORM_POSIX
+        // pthread_setname_np is a non-standard GNU extension.
+        char name_clamp[16];
+        strncpy(name_clamp, name, 15);
```

Comment:
Isn't strncat_s a C11 feature? Could I just put in a null check on `name` instead?

*looks at code*

No wait. I see what you did there :-)

Sure.